### PR TITLE
Added Whatsnew in nav drawer

### LINKED ISSUE #565: Add Whats New to navigation drawer
- We already have a whats new fragment which is shown once, when app is upgraded/downloaded
- Add this option in navigation drawer also ( in Extras)
- On clicking this, a new fragment should open up. It should have a list of all the whatsnews with the versions We keep it in the form of json : https://github.com/Swati4star/Images-to-PDF/blob/1688389c5ce71b5bea046b602142d1ad3eefe59f/app/src/main/assets/whatsnew.json

### PR DESCRIPTION
**Pull request by jddeep**
# Description
Added whatsnew in nav drawer.
Screenshot_20190323-094312

Screenshot_20190322-185641

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #565 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by jddeep** [comment]
@Swati4star Please review.

**Comment by jddeep** [comment]
> Changes look good. Just one change required. When `Whats New` is selected through navigation drawer, it should not remain selected. See line 485.

@sidhuparas can you specify the activity of which you are talking about the line `485`

**Comment by jddeep** [comment]
@sidhuparas Check it out now. Thanks.

**Comment by jddeep** [comment]
@sidhuparas Please merge. I need to work on another issue and make a PR.

**Comment by jddeep** [comment]
@Swati4star Please merge. I need to work on another issue and make a PR.

**Comment by vyankatesh24** [comment]
@jddeep  Add issue number in template.

**Comment by jddeep** [comment]
@vyankatesh24 It's done. Thanks.

File: app/src/main/java/swati4star/createpdf/activity/MainActivity.java [hunk 1]
```diff
@@ -479,7 +482,8 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
             e.printStackTrace();
         }
         // if help or share is clicked then return false, as we don't want them to be selected
```
Comment (vyankatesh24) [inline app/src/main/java/swati4star/createpdf/activity/MainActivity.java]: Update this comment.

**Comment by jddeep** [comment]
@vyankatesh24 All changes are done. Thanks. Please merge now. Cheers!

**Comment by vyankatesh24** [comment]
> @vyankatesh24 All changes are done. Thanks. Please merge now. Cheers!

Please update Screenshot!

**Comment by jddeep** [comment]
@vyankatesh24 Done!

**Comment by jddeep** [comment]
@vyankatesh24 Please merge. I need to work on another issue.

**Pull request merged**